### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by serentinos**
- [DEMO LINK](https://serentinos.github.io/layout_catalog/)
- [TEST REPORT LINK](https://serentinos.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

**Review by vakopov — APPROVED** [review]
👍  good job. nothing to add.

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,423 @@
...
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
```
Comment (Anna-Tkachenko) [inline src/index.html:40]: Move content to the new line for better readability

File: src/index.html [hunk 2]
```diff
@@ -1,12 +1,423 @@
...
+            <a href="#video" class="nav__link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products">
+      <div class="products__container">
+        <ul class="products__list">
+          <li class="products__item">
```
Comment (Anna-Tkachenko) [inline src/index.html:75]: Soo many containers and wrappers here
Let's get rid of them
`section` container with `article` for each card inside will be enough

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    border: 1px solid transparent;
+    border-radius: 5px;
+
+    transition: all 0.1s;
+
+    &:hover {
+      color: $color-blue-accent;
+
+      background-color: white;
+      border: 1px solid $color-blue-accent;
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/card.scss:95]: You should change only border-color on hover state

File: src/styles/blocks/nav.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
+.nav {
+  &__list {
+    display: flex;
+
```
Comment (Anna-Tkachenko) [inline src/styles/blocks/nav.scss:4]: delete redundant empty lines

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]

File: src/styles/blocks/nav.scss (hunk 4 above)
Comment (Anna-Tkachenko) [reply src/styles/blocks/nav.scss:4]: @serentinos Oh, I see, it's just for better readability
In this way you can leave it as it is, don't need to do any changes ⭐️

File: src/index.html (hunk 2 above)
Comment (serentinos) [reply src/index.html]: @Anna-Tkachenko I have tried to get rid of many containers, but after fixing two of the tests didn't pass. Can you help? @Anna-Tkachenko

Comment (Anna-Tkachenko) [reply src/index.html]: @serentinos I'll check it in next review 🐝

Comment (Anna-Tkachenko) [reply src/index.html]: @serentinos I didn't see any failed tests in the link in PR description. Did you re-deploy the tests?

File: src/index.html (hunk 1 above)
Comment (Anna-Tkachenko) [reply src/index.html:43]: You didn't fix the comment, move content to the new line

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,436 @@
...
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link"
+            >
```
Comment (Anna-Tkachenko) [inline src/index.html:58]: Check closed bracket here

File: src/index.html [hunk 6]
```diff
@@ -1,12 +1,436 @@
...
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link"
+            >
```
Comment (Anna-Tkachenko) [inline src/index.html:65]: Same here, check for all links

File: src/index.html [hunk 7]
```diff
@@ -1,12 +1,436 @@
...
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="products">
+      <ul class="products__list">
+        <li class="products__item">
```
Comment (Anna-Tkachenko) [inline src/index.html:89]: `ul` and `li` tags are redundant here
probably it's not the list, it's more section with separate block inside
so will be better to use `section` as container and `article` for each separate card

File: src/index.html [hunk 8]
```diff
@@ -1,12 +1,436 @@
...
+          <article class="card products__card" data-qa="card">
+            <img
+              src="./images/imac.jpeg"
+              alt="APPLE A1419 iMac 27inch Retina 5K Monoblock (MNED2UA/A)"
+              class="card__image"
+            >
+
+            <div class="card__info">
```
Comment (Anna-Tkachenko) [inline src/index.html:97]: Do you need this additional wrapper here? For what purpose? Can you get rid of it?

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Good job with css! Check some suggestions in HTML structure

**Review by Anna-Tkachenko — APPROVED** [review]
Well Done! ⭐️

**Pull request closed without merging**